### ISSUE TITLE
[dataset] The images of Diabetic Retinopathy

### ISSUE THREAD

**Issue by WeiQijie** [luogen1996/RepAdapter#15]
Hi, thanks for sharing the download link of VTAB-1k dataset. I have a question about the Diabetic Retinopathy dataset. As far as I know, the Diabetic Retinopathy dataset has 53576 test images while you have provided 42670 images for test. Could you please tell me where does the gap come from.
Thanks a lot.

**Comment by WeiQijie** [comment]
It seems that tensorflow_dataset put about 10000 images into validation set. Therefore, the number 42670 is correct for VTAB-1k.
